src/v2/Apps/Artist/Components/MarketStats.tsx

```diff
@@ -0,0 +1,217 @@
+        <Text variant="lg">Market Signals</Text>
+        <Text variant="lg" color="black60">
+          Averages over last 36 months
+        </Text>
+      </Box>
+
+      {/* Market Stats Values */}
+      <GridColumns gridRowGap={[2, 0]}>
+        <Column span={2}>
+          <Select
+            selected={selectedPriceInsight.medium || undefined}
+            options={mediumOptions.current}
+            title="Medium"
+            onSelect={selectedMedium => {
+              const priceInsight = priceInsights.find(
+                p => p.medium === selectedMedium
+              )
+              if (priceInsight) {
+                setSelectedPriceInsight(priceInsight)
+              }
+            }}
+          />
+        </Column>
+        <Column span={10}>
+          <GridColumns gridRowGap={[2, 0]}>
+            <Column span={6}>
+              <GridColumns gridRowGap={[2, 0]}>
+                <Column span={[6]}>
+                  <Text variant="text">Yearly lots sold</Text>
+                  <Text variant="xxl" data-test-id="annualLotsSold">
+                    {selectedPriceInsight.annualLotsSold}
+                  </Text>
+                </Column>
+
+                <Column span={[6]}>
+                  <Text variant="text">Sell-through rate</Text>
+                  <Text variant="xxl">{formattedSellThroughRate}%</Text>
+                </Column>
+              </GridColumns>
+            </Column>
+
+            <Column span={6}>
+              <GridColumns gridRowGap={[2, 0]}>
+                <Column span={[6]}>
+                  <Text variant="text">Sale price</Text>
+
+                  <Text variant="xxl">${formattedAverageValueSold}</Text>
+                </Column>
+
+                <Column span={[6]}>
+                  <Text variant="text">Sale price over estimate</Text>
+
+                  <Text variant="xxl">
+                    <Join separator={<Spacer mr={0.5} />}>
+                      {deltaIcon}
+                      {formattedMedianSaleOverEstimatePercentage}%
+                    </Join>
+                  </Text>
+                </Column>
+              </GridColumns>
+            </Column>
+          </GridColumns>
+        </Column>
+      </GridColumns>
+      <Spacer mt={12} />
+    </>
+  )
+}
+
+export const MarketStatsFragmentContainer = createFragmentContainer(
+  MarketStats,
+  {
+    priceInsightsConnection: graphql`
+      fragment MarketStats_priceInsightsConnection on PriceInsightConnection {
+        edges {
+          node {
+            medium
+            annualLotsSold
+            annualValueSoldCents
+            sellThroughRate
+            medianSaleOverEstimatePercentage
+          }
+        }
+      }
+    `,
+  }
+)
+
+export const MarketStatsQueryRenderer: React.FC<{
+  artistInternalID: string
+  environment: RelayModernEnvironment
+}> = ({ artistInternalID, environment }) => {
+  return (
+    // <QueryRenderer<MarketStatsQuery>
+    <QueryRenderer<any>
```

Comment:
We'll want to use `SystemQueryRenderer` here; the default relay query renderer will double-fetch on the server side render pass. 

But a bigger question is: why do we need a query renderer at all for this component?

Reply:
Great idea to create a danger rule for this! 🙌